build-elements/ff4j-dashboard.js

```diff
@@ -0,0 +1,15 @@
+const fs = require("fs-extra");
+const concat = require("concat");
+const del = require("del");
+
+(async function build() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function.](https://app.codacy.com/manual/paul58914080/ff4j-web-console/pullRequest?prid=5258623)